packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card-section.tsx

```diff
@@ -6,7 +6,7 @@ import { TJurisdictionCardSectionTitleIndicators } from 'Components/props.types'
 import classNames from 'classnames';
 
 const JurisdictionCardSection = ({ cardSectionItem }: TJurisdictionCardSectionProps) => {
-    function renderTitleIndicator(titleIndicators: TJurisdictionCardSectionTitleIndicators) {
+    const renderTitleIndicator=(titleIndicators: TJurisdictionCardSectionTitleIndicators) =>{
```

Comment:
@shaheer-deriv help to set `prettier - Code formatter` as your default formatter to help to fix issue as such.
`const renderTitleIndicator = (titleIndicators: TJurisdictionCardSectionTitleIndicators) => {`